source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,71 @@
+=======================================
+Accessing data of offline Users/Players 
+=======================================
+
+It may be necessary for plugins to access player data even when the player is offline.
+Since ``Sponge.getServer().getPlayer()`` does only return Players who are online one needs to find another solution.
+
+Some plugins store the relevant data themselves and associate the user by using ``Sponge.getServer().getGameProfileManager()``.
+But writing different code for offline and online users is not necessary:
+
+The Service Manager natively provides a service called ``UserStorageService`` which is capable of returning ``User`` instances for Players who are currently offline
```

Comment:
"The `ServiceManager` ... a service known as the `UserStorageService` ... for `Player`s who ..."

Essentially backticks on ServiceManager,
called -> known as the ... (called sounds unprofessional imo, if anything it'd be "called the"),
backticks on Player and then append \ and s. Ex: \`Player\`\\ s